src/category_theory/limits/limits.lean

```diff
@@ -616,16 +616,16 @@ def lim : (J ⥤ C) ⥤ C :=
 
 variables {F} {G : J ⥤ C} (α : F ⟶ G)
 
-@[simp, reassoc] lemma lim.map_π (j : J) : lim.map α ≫ limit.π G j = limit.π F j ≫ α.app j :=
+@[simp, reassoc] lemma limit.map_π (j : J) : lim.map α ≫ limit.π G j = limit.π F j ≫ α.app j :=
```

Comment:
I think the reason it was called `lim.map_pi` is that the statement starts with `lim.map \alpha ...`. Hence it follows the naming conventions, and I'm inclined to leave it the way it was...

Reply:
In that case, should `limit.map_pre` be renamed to `lim.map_pre` by the same argument?

Reply:
@TwoFX Fair enough, I agree with the confusion. I think that `limit.map_pre` is actually the one that is "violating" the naming convention. If you want to PR those changes, that would be nice.